app/views/console/comps/header.phtml

```diff
@@ -211,7 +211,7 @@
             required
             maxlength="36"
             class=""
-            pattern="^[a-zA-Z0-9][a-zA-Z0-9_.-]{1,36}$"
+            pattern="^[a-zA-Z0-9][a-zA-Z0-9._-]{0,35}$"
```

Comment:
Is there a difference between 
`[a-zA-Z0-9_.-]` and `[a-zA-Z0-9._-]`

Reply:
There isn't, just for consistency. It was reversed only in one place, so I fixed it.

![CleanShot 2022-03-23 at 08 42 56](https://user-images.githubusercontent.com/19310830/159648217-e80b654d-7ea9-4abb-bb93-255683d7315f.png)